jackson-core/src/main/java/io/micronaut/jackson/core/tree/TreeGenerator.java

```diff
@@ -167,7 +167,7 @@ public JsonNode getCompletedValue() {
     }
 
     @Override
-    public void writeStartArray() throws IOException {
+    public void writeStartArray() {
```

Comment:
why removing the `throws`. Does not throw IOException anymore?

Reply:
You don't need to write exceptions from interface if your implementation don't throw them